### ISSUE TITLE
[XR Plugin Management] AndroidManifest.xml not found in unityLibrary/AndroidManifest.xml

### ISSUE THREAD

**Issue by itssidhere** [juicycleff/flutter-unity-view-widget#801]
**Describe the bug**
When I enable Ar core in my project and try to export using the plugin. I think unity distributes the AndroidManifest.xml and there is not AndroidManifest.xml in the unityLibrary folder so the plugin throws a file not found exception

**To Reproduce**
1. Enable AR core functionality by import Ar Foundation and Ar core google
2. Create basic AR scene
3. export in debug mode

**Screenshots**
![image](https://user-images.githubusercontent.com/16454736/235252358-8b6c7113-a6aa-4a04-97be-e08de99c799c.png)

**Unity (please complete the following information):**
 - OS: Windows 11
 - Version 2022.2

**Additional context**
The project successfully builds when building from Unity but cannot be exported!

**Comment by kemalbayikk** [comment]
I am facing with the same issue. Any updates?

**Comment by guoguoguilai** [comment]
The same issue. How to solve this?

**Comment by timbotimbo** [comment]
Looks like Unity broke their own code.

Based on info from [this unity forum topic](https://forum.unity.com/threads/androidmanifest-missing-from-unitylibrary-src-main-after-project-export-for-gradle-build.1417725/).
The XR Plugin Management package included [a change](https://docs.unity3d.com/Packages/com.unity.xr.management@4.3/changelog/CHANGELOG.html#431---2022-10-14) in 4.3.1.
> [4.3.1] - 2022-10-14
> Clean up AndroidManifest.xml every XR build to prevent conflicts with other configurations.

Was reported to Unity [here](https://forum.unity.com/threads/in-40099-xr-management-package-4-3-1-deletes-unitylibrary-androidmanifest-xml.1432444/), and on may 5th it was said to be fixed in the next version.

###
So in short, downgrade XR Plugin Management to 4.2.2 or lower.
Or wait for Unity to release a newer version like 4.3.4.

**Comment by kemalbayikk** [comment]
Downgrading Unity version to 2021.3.22f1 solved issue for me

**Comment by Kashif-Asghar60** [comment]
I am experiencing the same error. As a newcomer to the Unity environment, I was following a 4-year-old YouTube tutorial. However, when I attempt to export for Android (debug/release), I encounter an error stating that the AndroidManifest file cannot be found. Unfortunately, I am unsure how to resolve this issue.

After receiving a suggestion from @kemalbayikk, I downgraded to version 2021.3.22f1, but the error persists. 
I'm using demoapp (cube) project in unity.
<img width="919" alt="image" src="https://github.com/juicycleff/flutter-unity-view-widget/assets/103773636/1d7bce38-76ed-4fbf-ab6f-5590f05ffde6">

**Comment by Ahmadre** [comment]
> Looks like Unity broke their own code.
> 
> Based on info from [this unity forum topic](https://forum.unity.com/threads/androidmanifest-missing-from-unitylibrary-src-main-after-project-export-for-gradle-build.1417725/). The XR Plugin Management package included [a change](https://docs.unity3d.com/Packages/com.unity.xr.management@4.3/changelog/CHANGELOG.html#431---2022-10-14) in 4.3.1.
> 
> > [4.3.1] - 2022-10-14
> > Clean up AndroidManifest.xml every XR build to prevent conflicts with other configurations.
> 
> Was reported to Unity [here](https://forum.unity.com/threads/in-40099-xr-management-package-4-3-1-deletes-unitylibrary-androidmanifest-xml.1432444/), and on may 5th it was said to be fixed in the next version.
> 
> So in short, downgrade XR Plugin Management to 4.2.2 or lower. Or wait for Unity to release a newer version like 4.3.4.

Thanks Timbo. I faced the same problem. Maybe we should notice these somehow in Changelog or so (if we for example tackle 2022.2.0.)

**Comment by timbotimbo** [comment]
I Added a note to the Readme and removed the package from the example project, as the example doesn't use AR ayway.

**Comment by karolszczesniakk** [comment]
Upgrading XR Plugin Management to 4.4.0 fixed the problem for me.
![image](https://github.com/juicycleff/flutter-unity-view-widget/assets/80595109/d25a13ce-6a05-4c19-9c18-14fb6a836859)
